src/frontend/src/lib/components/ui/ButtonIcon.svelte

```diff
@@ -5,14 +5,12 @@
 </script>
 
 <button
-	class="icon text-primary flex flex-col text-center text-xs font-normal"
+	class="tertiary-link icon flex flex-col text-center text-xs font-normal"
```

Comment:
Curiosity question, why not just calling it "tertiary"? Why "-link"?

Reply:
According to plans and Figma, there is a "tertiary" class

Reply:
Different than `tertiary-link`? I know it's minimal, just feel like that adding a suffix here feels like already adding an exception you know what I mean?

Reply:
understood! well the difference is really the color set ultimately

Do you prefer that I create a class `tertiary` with a sub class `link`

Reply:
Oh yes, I would rather like that if it makes sense to you too?

Reply:
yes yes., done! have a look